integration-tests/contracts/contract_deployer.go

```diff
@@ -191,14 +192,14 @@ func (e *EthereumContractDeployer) DeployReadAccessController() (ReadAccessContr
 		auth *bind.TransactOpts,
 		backend bind.ContractBackend,
 	) (common.Address, *types.Transaction, interface{}, error) {
-		return ethereum.DeploySimpleReadAccessController(auth, backend)
+		return simple_read_access_controller.DeploySimpleReadAccessController(auth, backend)
 	})
 	if err != nil {
 		return nil, err
 	}
 	return &EthereumReadAccessController{
 		client:  e.client,
-		rac:     instance.(*ethereum.SimpleReadAccessController),
+		rac:     instance.(*simple_read_access_controller.SimpleReadAccessController),
```

Comment:
Is it being used in any test currently?

Reply:
Same question. We might be able to rip this one out entirely.

Reply:
Oh that is a good question, no usages in chainlink at least. Will dig through the other repos.

Reply:
Ripped out and cleaned up the PR. Good call out!